index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/db139aff16.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="main.js" defer></script>
+    <script src="services.js" defer></script>
+    <script src="enable.js" defer></script>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/css/bootstrap.min.css"
+        integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+
+</head>
+
+<body>
+    <section class="front-page" id="frontPage">
+
+        <nav class="top-nav">
+            <nav class="desktop-navbar background">
+                <span class="fa-brands fa-facebook-f"></span>
+                <span class="fa-brands fa-twitter"></span>
+                <h5> Blog </h5>
+                <h5> Sponsor </h5>
+                <h5>News </h5>
+
+            </nav>
+            <div class="desktop-nav">
+                <button type="button" class="ham-burger" id="desk">
+                    <span class="glyphicon glyphicon-menu-hamburger"></span>
+                </button>
+                <a href="#frontPage">
+                    <img class="desktop-logo" src="images/logo.jpeg" alt="Logo Image">
+                </a>
+                <div class="desktop-links">
+                    <a href="#frontPage" class=" link" id="red">Home</a>
+                    <a href="#mainPage" class="link" id="mid">About</a>
+                    <a href="" class=" link border" id="red">Contact Us</a>
+                </div>
```

Comment:
- [ ] I would kindly suggest you increase your nav links to replicate exactly the number of nav links we have in the [Cindy Shin design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

Your design 👇 

![link](https://user-images.githubusercontent.com/78634841/210978667-0143dedc-b133-4781-983e-e32fb8aa87dc.png)

Cindy Shin design 👇 

![link1](https://user-images.githubusercontent.com/78634841/210978628-135ea76c-4e8c-49e1-a0b7-b551e47a2192.png)